modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -208,6 +208,25 @@ ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:bond0.100:none
 bond=bond0:em1,em2:mode=active-backup
 vlan=bond0.100:bond0
 ----
+
+a|Optional: Network teaming can be used as an alternative to bonding by using the `team=` parameter. In this example:
+
+* The syntax for configuring a team interface is: `team=name[:network_interfaces]`
++
+_name_ is the team device name (`team0`) and _network_interfaces_ represents a comma-separated list of physical (ethernet) interfaces (`em1, em2`).
+
+[NOTE]
+====
+Teaming is planned to be deprecated when {op-system} switches to an upcoming version of {op-system-base}. Please see the following https://access.redhat.com/solutions/6509691[Knowledgebase Article] for more information.
```

Comment:
```suggestion
Teaming is planned to be deprecated when {op-system} switches to an upcoming version of {op-system-base}. For more information, see https://access.redhat.com/solutions/6509691[Knowledgebase Article].
```

Minimalism.

Reply:
thanks @sagidlow ! I've made an update in 8f75adc...0117080, let me know what you think! I was debating whether to leave the link url as is with no text, but then settled on having the link text be "Red Hat Knowledgebase Article" since it's used elsewhere.